magic_marker/cli.py

```diff
@@ -0,0 +1,41 @@
+# -*- coding: utf-8 -*-
+
+"""Console script for magic-marker."""
+# ======================================================================================================================
+# Imports
+# ======================================================================================================================
+from __future__ import absolute_import
+from magic_marker.magic_marker import MagicMarker
+import sys
+import click
+
+
+# ======================================================================================================================
+# Main
+# ======================================================================================================================
+@click.command()
+@click.argument('test_path', type=click.Path(exists=True))
+@click.argument('pytest_mark_name', type=click.STRING)
+def main(test_path, pytest_mark_name):
+    """Automatically fix tests that are not marked with a UUID.
+
+    \b
+    Required Arguments:
+        test_path               the path to pass to flake8
+        pytest_mark_name        ensure this mark is present and marked with a UUID
+    """
+
+    try:
+        w = MagicMarker(directory=test_path, mark=pytest_mark_name)
```

Comment:
`w` as a variable for the MagicMarker instance may no longer make sense.

Reply:
WagicWarker

Reply:
fixed